### DISCUSSION TITLE
Changed computers and cannot find where to update new tailwind.config location path

### DISCUSSION THREAD

**Discussion by S-man23** [tailwindlabs/tailwindcss#9985] [Help]
Hi,

I have recently changed computers and have pulled my project from GitHub to my new laptop. Tailwindcss doesn't compile or work on my new computer as it's looking for the old file path from my previous computer. The tailwind.config file exists in the project and I have attempted to include the new path in that folder, but no matter what I try, it doesn't work.

The error message I receive when trying to run/compile tailwinds is:

"CssSyntaxError: /Users/louis/Desktop/CodeProjects/nput.css:1:1: The config file at "./tailwindcss-config.js" does not exist. Make sure the path is correct and the file exists."

I've tried uninstalling and reinstalling tailwind, but it doesn't change the file location either. Where do I go and amend this path so I can use tailwindcss for this project on my new laptop?

Thanks!

**Comment by wongjn** [comment]
You say that the <q>The tailwind.config file exists in the project</q> but the error message mentions `tailwindcss-config.js` – a dash instead of a dot delimiter. Perhaps a text search in your project for any references to `tailwindcss-config.js` may help?

**Reply by S-man23** [reply]
I just did a search and nothing appears where I can edit the location path

**Reply by wongjn** [reply]
Did you also do a search with `tailwindcss-config` (without the extension)? Otherwise, if you can provide a remotely-hosted git repository, we can take a closer look for you.

**Reply by S-man23** [reply]
yeah, I tried that and these are the results from the search 
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/63434043/205749806-b39e0670-0f9e-4649-8970-75c7bb35a646.png">

Here's a screenshot of me trying to compile tailwinds and the error message I receive 
<img width="505" alt="image" src="https://user-images.githubusercontent.com/63434043/205750054-db246702-14a9-44aa-a27d-d73b783af016.png">

What is your GitHub username? I'll add you to the project for you to take a look

**Reply by wongjn** [reply]
You can add me with this user to the project.

**Reply by wongjn** [reply]
It seems like the `@config` lines that you showed could be the culprits – you indeed have no `tailwindcss-config.js` file in your project. You could try removing the lines altogether, since it seems like you're using the default `tailwind.config.js` file so the `@config` directive is redundant.

**Reply by S-man23** [reply]
Brilliant! That worked. Thanks heaps!
